core/utils/npm.py

```diff
@@ -65,3 +65,12 @@ def uninstall(package, option='', folder=None):
     @staticmethod
     def get_version(package):
         return Npm.__run_npm_command('show {0} version'.format(package))
+
+    @staticmethod
```

Comment:
I think this method is the same as __run_npm_command(). Why not make __run_npm_command public and use it instead of duplicating code.

Reply:
Agree with this. I'm going to update it and add a new commit to the PR.